src/test/java/org/jenkinsci/plugins/ghprb/GhprbRepositoryTest.java

```diff
@@ -109,6 +109,11 @@ public void setUp() throws Exception {
         doReturn(gt).when(trigger).getGitHub();
         
         given(gt.getRepository(anyString())).willReturn(ghRepository);
+
+        // Set rate limit before starting trigger
+        GHRateLimit initRateLimit = new GHRateLimit();
```

Comment:
Actually on second look it's the repoTimes that needs to be incremented by one if you associate the mock objects earlier.